WISE-Community/WISE-Client
# Show latest posts from Discourse news category on site home

## Changes
Added a latest news component that displays links to the 3 latest topics in a specified Discourse category. Latest news is displayed inside the HeroSection on the site home. HeroSection was modified to display optional side content.

Also replaced all "news" links on the site home, menus, and footer with links to the WISE Community (Discourse).

Closes #220. 

## Test
- Load the site home and make sure the last 3 posts from https://wise-discuss.berkeley.edu/c/news-announcements/7 are shown and that clicking on each takes you to the post.
- Test on different screen sizes to make sure layout looks good and all the Hero content is readable.
- Also ensure that the "More news" link takes you to the Discourse category.
- Check that all the "Community" links work in the menus, footer, and Community call-to-action section at the bottom of the site home.
